man-roxygen/param_test_server.R

```diff
@@ -0,0 +1,2 @@
+#' @param test_server (`character(1)`)\cr
+#' Whether to use the OpenML [test server](OpenML server to be used. Defaults to the public OpenML server seen on  https://openml.org).
```

Comment:
is this valid markdown?

Reply:
no, already corrected in the next commit